dart/ooniengine/pubspec.lock

```diff
@@ -0,0 +1,145 @@
+# Generated by pub
```

Comment:
perhaps you want to ignore this one too?

Reply:
From https://dart.dev/tools/pub/glossary#lockfile

> A file named pubspec.lock that specifies the concrete versions and other identifying information for every immediate and transitive dependency a package relies on.
>
> Unlike the pubspec, which only lists immediate dependencies and allows version ranges, the lock file comprehensively pins down the entire dependency graph to specific versions of packages. A lockfile ensures that you can recreate the exact configuration of packages used by an application.
>
> The lockfile is generated automatically for you by pub when you run [pub get](https://dart.dev/tools/pub/cmd/pub-get), [pub upgrade](https://dart.dev/tools/pub/cmd/pub-upgrade), or [pub downgrade](https://dart.dev/tools/pub/cmd/pub-downgrade). If your package is an application package, you will typically check this into source control. For library packages, you usually won’t.

Because this is mostly a library, I'd say we want to commit it.